.github/workflows/archive/documentationcheck.py

```
@@ -0,0 +1,222 @@
+
+    Returns:
+        None
+
+    """
+    # Parsing the command line arguments
+    args = _arg_parser_resolver()
+    # Getting the git repo
+    repo_feature = git.Repo(args.directory)
+    (_, repository_directory) = args.repository.split("/")
+    repo_merge = git.Repo.clone_from(
+        "https://github.com/{}.git".format(args.repository),
```

Ruff (0.11.9)

Use f-string instead of `format` call

Convert to f-string

(UP032)